src/components/ListView.js

```diff
@@ -1,11 +1,33 @@
-import React from 'react';
-
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 import { v4 as uuidv4 } from 'uuid';
 
 import ListItem from '../components/ListItem';
 
 const ListView = ({ shoppingList, loading, error }) => {
-  return (
+  const [shoppingListEmpty, setShoppingListEmpty] = useState(true);
+  const [length, setLength] = useState(0);
+
+  useEffect(() => {
+    if (loading === false && shoppingList[0] !== undefined) {
+      setLength(shoppingList[0].items.length);
+    }
+
+    if (length >= 1) {
+      setShoppingListEmpty(false);
+    } else {
+      setShoppingListEmpty(true);
+    }
+  }, [length, loading, shoppingList]);
+
+  return loading === false && shoppingListEmpty ? (
+    <div className="prompt">
+      <h1>No Item Added</h1>
+      <Link to="/add-view" className="add-button">
+        Add Item
+      </Link>
+    </div>
```

Comment:
Looks great! That's definitely how I would do it 👍

Reply:
Thank you Alex!